shared/src/types.rs

```diff
@@ -765,6 +765,12 @@ pub struct State {
     pub cidrs: Vec<Cidr>,
 }
 
+/// This model is sent as a response to the /info endpoint.
+#[derive(Debug, Default, Clone, Deserialize, Serialize)]
+pub struct Info {
+    pub override_endpoint_with_unspecified_ip_is_supported: bool,
```

Comment:
Nit: add `#[serde(default)]` to cope with the very hypothetical situation the field is removed (false is a good default).